.Lib9c.Tests/Action/Buy3Test.cs

```diff
@@ -114,18 +126,23 @@ public Buy3Test(ITestOutputHelper outputHelper)
                 .SetState(_sellerAvatarAddress, sellerAvatarState.Serialize())
                 .SetState(_buyerAgentAddress, buyerAgentState.Serialize())
                 .SetState(_buyerAvatarAddress, _buyerAvatarState.Serialize())
-                .MintAsset(_buyerAgentAddress, _goldCurrencyState.Currency * 100);
+                .MintAsset(_buyerAgentAddress, _goldCurrencyState.Currency * 300);
```

Comment:
Why multiply 300 in here?

Reply:
@riemannulus The price of each store item is 100 NCG. We had to do a total of three purchase tests, so we prepared 300NCG. But I modified it because it is hard coding.